### ISSUE TITLE
where is the .env file?

### ISSUE THREAD

**Issue by ndonnipa** [eahefnawy/awsm-mailer#5]
i cant seem to find the .env file?

**Comment by marcogbarcellos** [comment]
@ndonnipa take a look on this project here: https://github.com/motdotla/dotenv .
It'll clarify on how to use and set ENV Variables throughs this .env file.

**Comment by rcfrias** [comment]
I think the description of this project must be updated, at least to reference that link you mentioned before @marcogbarcellos .  It is not clear if the owner forgot to add this file, or in fact he assumed we all be heading to the [https://github.com/motdotla/dotenv](https://github.com/motdotla/dotenv) library.

This post explains the situation in an easier context:  [https://medium.com/@rafaelvidaurre/managing-environment-variables-in-node-js-2cb45a55195f#.pfdp7ld1c](https://medium.com/@rafaelvidaurre/managing-environment-variables-in-node-js-2cb45a55195f#.pfdp7ld1c)

**Comment by rcfrias** [comment]
I think the missing info is:  "Create a file named .env"  <- this file gets included in the deploy and its automatically used to authenticate with the Email Server.  No need to load or install any other dependency. I just tested it.  Also, the docs are inaccurate, the pull request is valid for the right command `serverless invoke -f send -p event.json` but @eahefnawy hasn't merged this yet. I think he is not actively checking this project.

**Comment by TheDagger** [comment]
Agreed, it needs to be updated the documentation... I had no idea about using -p instead of -e
